packages/venia-ui/lib/tokens.css

```diff
@@ -85,6 +85,8 @@
 
     /* dimensions */
     --venia-global-maxWidth: 1440px;
+    --venia-global-header-minHeight: 80px;
+    --venia-mobile-global-header-minHeight: 56px;
```

Comment:
I am not sure how we name mobile-specific tokens. Team, what do your guys think?

Reply:
We discussed this in dev time, and if I recall correctly you could use a single variable, `--venia-global-header-minHeight` and change it here in `tokens.css` using a media query. so that you don't need a media query in `scrollAnchor.css` or two variable names.

```
@media (max-width: 960px) {
    :global(:root) {
        --venia-global-header-minHeight: 56px;   
    }
}  
```

Reply:
Also it looks like we might need three media queries as there are three different heights of the header, at least from my brief inspection:

[![Image from Gyazo](https://i.gyazo.com/378804d88abf29d49368473e94d7847b.gif)](https://gyazo.com/378804d88abf29d49368473e94d7847b)